app/src/main/java/com/github/sdp/tarjetakuna/ui/home/HomeFragment.kt

```diff
@@ -55,7 +66,36 @@ class HomeFragment : Fragment() {
 
         val addRandomCardButton: Button = binding.addRandomCardButton
         addRandomCardButton.setOnClickListener {
-            homeViewModel.addRandomCard()
+            //TODO : Remove the comment and the next lines when the branch 80 is ready to be merged
```

Comment:
Even if its temporary, maybe it's useful to copy the CommonMagicCards/Set classes from the tests to the utils package in main package for the future as it seems we're always in need of a temporary card while the parts we need are not merged yet.

Reply:
In that part of the code, it will be removed before the merge anyway, so I don't think that's helpful.